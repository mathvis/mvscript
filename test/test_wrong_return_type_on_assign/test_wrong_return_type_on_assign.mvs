func a() float {
	return 5.0;
}
let b: int;
b = a();
