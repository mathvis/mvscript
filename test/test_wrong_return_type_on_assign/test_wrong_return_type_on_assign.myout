error: Expected IntT but got FloatT
 --> test/test_wrong_return_type_on_assign/test_wrong_return_type_on_assign.mvs:3:1
  |
3 | b = a();
  | ^
  |
hint: Consider changing the variable type or declaring a new variable.
