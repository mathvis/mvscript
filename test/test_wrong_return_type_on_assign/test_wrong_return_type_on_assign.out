error: Expected IntT but got FloatT
 --> test/test_wrong_return_type_on_assign/test_wrong_return_type_on_assign.mvs:5:1
  |
5 | b = a();
  | ^
  |
hint: Consider changing the variable type or declaring a new variable.
